lib/tls.js

```diff
@@ -221,11 +221,12 @@ exports.checkServerIdentity = function checkServerIdentity(host, cert) {
         valid = wildcard(cn);
 
       if (!valid)
-        reason = `Host: ${host}. is not cert's CN: ${cn}`;
+        reason = `Host: ${hostname}. is not cert's CN: ${cn}`;
```

Comment:
Nit: should we also change the message prefix from `Host:` to `Hostname:`? Or is this not done because it would be a server-major change?

Reply:
@lpinca Hard to say. It's an internal error with a code (`ERR_TLS_CERT_ALTNAME_INVALID`). So it should be OK, but I don't know that we ever officially decided that message changes are no longer breaking changes in those cases. So could go either way. Maybe make that change in a separate PR just to make sure any debate about it doesn't delay the rest of these changes? 

Additionally, the period seems to be an error and should be removed here and in line 229.